src/theme/globalStyle.tsx

```diff
@@ -26,7 +26,7 @@ export const GlobalStyle = createGlobalStyle<{ theme: any }>`
 
 export const ThemedGlobalStyle = createGlobalStyle`
   html {
-    color: ${({ theme }) => theme.text1};
+    color: ${({ theme }) => theme.text4};
```

Comment:
walletconnect modal take the `body>color` style, I didn't find another way to change the `text color`, and it does not seem to affect anything, what do you think @alongoni?